composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "ayanozturk/phprest",
+    "name": "phprest/phprest",
```

Comment:
reverting this to original @adammbalogh 

previously I was aiming to publish these changes as a separate package.